### PR TITLE
Issue #15: Resolved

### PR DESCRIPTION
**Pull request by Jainil2004**
Issue #15 : Create a PR template 

Issue RESOLVED? YES

Description
Created a highly detailed Pull request template to streamline the pull request submission for the project.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+please follow the PULL REQUEST TEMPLATE pattern for generating pull requests
```
Comment (GuiBibeau) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Let's make this a comment.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- [ ] change 2
+- ...
+
+## DESCRIPTION OF THE CHANGES MADE
+<!-- provide a detailed explanation of the changes made -->
+- [ ] change 1:
+- description of change 1
+
+- [ ] change 2:
+- description of change 2
```
Comment (GuiBibeau) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: This can be removed since we have already a section for changes made and a description above.

**Review by GuiBibeau — COMMENTED** [review]
Thank you very much for your contribution! I left a few comments around making the template simpler.

The direction you took was great, just need a bit less things.

**Comment by Jainil2004** [comment]
Thanks for the review I really appreciate it. I have taken note of all the changes to be made and will report back to you once I am done with the changes. 
Thank you

**Comment by Jainil2004** [comment]
@GuiBibeau I have made all the suggested/requested changes in the template. please have a look and let me know if something feels out of the place in the template. 
Thank you

**Review by GuiBibeau — APPROVED** [review]

**Pull request merged**